docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,560 @@
+---
+authors:
+- reswqa:
+  name: "Weijie Guo"
+  twitter: "WeijieGuo12"
+- 1996fanrui:
+  name: "Rui Fan"
+  twitter: "1996fanrui"
+
+date: "2024-07-15T08:00:00Z"
+subtitle: ""
+title: Announcing the Release of Apache Flink 1.20
+aliases:
+- /news/2024/07/15/release-1.20.0.html
+---
+
+The Apache Flink PMC is pleased to announce the release of Apache Flink 1.20.0. As usual, we are
+looking at a packed release with a wide variety of improvements and new features. Overall, 142
+people contributed to this release completing 13 FLIPs and 300+ issues. Thank you!
+
+Let's dive into the highlights.
+
+# Standing on the Eve of Apache Flink 2.0
+
+Flink 1.0 was released seven years ago. Since several months, the community is actively planning and taking steps towards
+the next major release. The new 1.20 release is planned to be the last minor release before Flink 2.0, which is anticipated by the end of 2024.
+
+Start from Flink 1.19, the community has decided to officially deprecate multiple APIs that were approaching 
+end of life for a while. In 1.20, we further sorted through all relevant APIs that might need to be replaced
+or deprecated to clear the way for the 2.0 release:
+- Configuration Improvements: As Flink moves towards version 2.0, we have revisited all runtime & Table API/SQL 
+configuration options and identified several opportunities to enhance user-friendliness and maintainability.
+- Deprecate the Legacy [SinkFunction](https://nightlies.apache.org/flink/flink-docs-release-1.20/api/java/org/apache/flink/streaming/api/functions/sink/SinkFunction.html) API: Since its introduction in Flink 1.12, the [Unified Sink API](https://nightlies.apache.org/flink/flink-docs-release-1.20/api/java/org/apache/flink/api/connector/sink2/Sink.html)
+has undergone extensive development and testing. Over multiple release cycles, the API has demonstrated
+stability and robustness, aligning with the criteria set forth in [FLIP-197](https://cwiki.apache.org/confluence/display/FLINK/FLIP-197%3A+API+stability+graduation+process) for API stability graduation.
```

Comment:
I'd suggest providing the link from the Share button, instead of the link copied from the browser URL bar, which may be broken when the title is renamed.